.gitignore

```diff
@@ -28,6 +28,9 @@ xcuserdata
 build
 builtFramework
 AWS*.framework
+node_modules
+package.json
+package-lock.json
```

Comment:
i wonder where these came from, i also see the package.json file added in these commits

Reply:
Ack! I didn't mean to add the package.json file--I'll remove it.

I added this explicitly so we can use NPM-based build tools (like [standard-version](https://www.npmjs.com/package/standard-version)) to help with CI/CD. We may end up refactoring that out.